guides/plugins/plugins/administration/the-shopware-object.md

```diff
@@ -47,25 +47,32 @@ Learn more about them here: [Creating administration module](add-custom-module.m
 
-| Property | Description |
-| :--- | :--- |
-| ApiService | Registry which holds services to fetch data from the api |
-| Component | A registry for VueJS `components` |
-| Context | A set of contexts for the `app` and the `api` |
-| Defaults | A collection of default values |
-| Directive | A registry for [VueJS `directives`](https://vuejs.org/v2/guide/custom-directive.html) |
-| Filter | A registry for [VueJS template `filters`](https://vuejs.org/v2/guide/filters.html) |
-| Helper | A collection of helpers, e.g. the `DeviceHelper` where you can listen on the `resize` event |
-| Locale | A registry for `locales` |
-| Mixin | A registry for `mixins` |
-| Module | A registry for `modules` |
-| Plugin | An interface to add `promise`based hooks to run when the administration launches |
-| Service | A helper to get quick access to service, e.g. `Shopware.Service('snippetService')` |
-| Shortcut | A registry for keyboard shortcuts |
-| State | A wrapper for the [VueX](https://vuex.vuejs.org/) store to manage state |
-| Utils | A collection of utility methods like `createId` |
+| Property   | Description                                                                                 |
+| :--------- | :------------------------------------------------------------------------------------------ |
+| ApiService | Registry which holds services to fetch data from the api                                    |
+| Component  | A registry for VueJS `components`                                                           |
+| Context    | A set of contexts for the `app` and the `api`                                               |
+| Defaults   | A collection of default values                                                              |
+| Directive  | A registry for [VueJS `directives`](https://vuejs.org/v2/guide/custom-directive.html)       |
+| Filter     | A registry for [VueJS template `filters`](https://vuejs.org/v2/guide/filters.html)          |
+| Helper     | A collection of helpers, e.g. the `DeviceHelper` where you can listen on the `resize` event |
+| Locale     | A registry for `locales`                                                                    |
+| Mixin      | A registry for `mixins`                                                                     |
+| Module     | A registry for `modules`                                                                    |
+| Plugin     | An interface to add `promise`based hooks to run when the administration launches            |
+| Service    | A helper to get quick access to service, e.g. `Shopware.Service('snippetService')`          |
+| Shortcut   | A registry for keyboard shortcuts                                                           |
+| State      | A wrapper for the [VueX](https://vuex.vuejs.org/) store to manage state                     |
+| Utils      | A collection of utility methods like `createId`                                             |
+
+## TypeScript declarations
+
```

Comment:
Add a hint on when this is available (SW version)

```
{% hint style="info" %}
TypeScript declarations are available from Shopware Version 6.4.? onwards.
{% endhint %}
```

Reply:
Done, thank your for your support. It'll be part of the next minor release therefore 6.4.4.0